dftimewolf/lib/module.py

```diff
@@ -3,6 +3,7 @@
 
 import abc
 import logging
+from logging import handlers
```

Comment:
Why import this again? Just do logging.handlers.

Reply:
I'd love to, but doing so caused some actual runtime errors: https://github.com/log2timeline/dftimewolf/pull/306

Reply:
Hmm, that's weird - please add a comment explaining that